### ISSUE TITLE
View types

### ISSUE THREAD

**Issue by KennyGoers** [Faltenreich/SkeletonLayout#6]
Is there a way to hook into the skeleton view factory? It would be nice to add or exclude view types (for example I have a MPAndroidChat)

**Comment by Faltenreich** [comment]
Good idea! I guess that currently the whole chart is masked and you would like to mask only certain sub-elements of the chart, am I right?

**Comment by KennyGoers** [comment]
Actually at the time of showSkeleton() its a ViewGroup with no children so it doesn't show. It comes out better as an entire block. For the time being I just put the Chart in a Framelayout with a View.

**Comment by Faltenreich** [comment]
I understand. You idea of an option to in- or exclude certain Views and ViewGroups makes much sense and I will think of an implementation in the near future. Of course you are free to submit a pull request which I would gladly review if you are in a hurry.

**Comment by vvictor10** [comment]
@Faltenreich Nice work on the library. Any updates on this issue. I have a loading state screen, where I need to exclude certain views from being masked and show the actual content on them. Would be great to specify a boolean to exclude them from the masking process.  Thanks
